drafts/2019-11-06-1900011.adoc

```diff
@@ -16,7 +16,7 @@ Give it a try in link:/about/[Open Liberty] 19.0.0.11.
 
 In Open Liberty 19.0.0.11:
 
-* <<link,<links>>>
+* <<pelt,Printing elapsed time message on completion of slow request with Request Timing 1.0>>
```

Comment:
Make the heading part of this match the titles etc above: "Diagnose slow requests more easily with improved request logging (Request Timing 1.0)" (but include the feature name in brackets here). And update the heading that this links to below.